python/python_direct/python_translate.cpp

```diff
@@ -1048,6 +1048,40 @@ class PythonTranslator : public OptInConstDispatch {
          sdpa_fwd_op->philox_offset()});
   }
 
+  // Map SdpaBwdOp to SdpaBwdOpRecord
+  void handle(const SdpaBwdOp* sdpa_bwd_op) final {
+    NVF_ERROR(sdpa_bwd_op != nullptr);
+
+    static const auto default_args = std::make_tuple(
+        KeywordArgument<Val*>{"dropout_p", nullptr},
+        KeywordArgument<Val*>{"is_causal", nullptr},
+        KeywordArgument<TensorView*>{"philox_seed", std::nullopt},
+        KeywordArgument<TensorView*>{"philox_offset", std::nullopt},
```

Comment:
why are we passing `std::nullopt` as default to a pointer type?

Reply:
does the API actually accept having philox_seed or philox_offset as nullopt? looking at c++ API doesn't seem to be the case.